lib/hydra/works/models/concerns/collection_behavior.rb

```diff
@@ -21,6 +21,15 @@ module CollectionBehavior
       type [Hydra::PCDM::Vocab::PCDMTerms.Collection, Vocab::WorksTerms.Collection]
     end
 
+    module ClassMethods
+      def type_validator
+        Hydra::PCDM::Validators::CompositeValidator.new(
```

Comment:
This is where it's messed up. I tried following the pattern I saw in other behaviors (using `@type_validator ||=`, but something is eluding me here).